sdk/core/azure-core/src/main/java/com/azure/core/util/UserAgentUtil.java

```diff
@@ -75,6 +78,20 @@ public static String toUserAgentString(String applicationId, String sdkName, Str
         return userAgentBuilder.toString();
     }
 
+    /**
+     * Removes azure- prefix from the SDK name in accordance with the
+     * <a href="https://azure.github.io/azure-sdk/general_azurecore.html#telemetry-policy">User Agent format guidance</a>.
+     *
+     * @param sdkName The original SDK name.
+     * @return The updated SDK name without the azure- prefix.
+     */
+    private static String standardizeSdkName(String sdkName) {
+        if (CoreUtils.isNullOrEmpty(sdkName)) {
+            return sdkName;
+        }
+        return AZURE_PREFIX.matcher(sdkName).replaceFirst("");
```

Comment:
Continuing with the comment that we always check for the same String, so `startsWith` and `substring` should offer better performance.